dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java

```diff
@@ -153,7 +152,7 @@ public void testCheckLinkageErrorMissingInterfaceMethodAt_interfaceAndClassSepar
         linkageChecker.checkLinkageErrorMissingMethodAt(methodSymbolReference);
 
     Truth8.assertThat(errorFound).isPresent();
-    Truth.assertThat(errorFound.get().getReason()).isEqualTo(Reason.INCOMPATIBLE_CLASS_CHANGE);
+    Truth.assertThat(errorFound.get().getReason()).isEqualTo(ErrorType.INCOMPATIBLE_CLASS_CHANGE);
```

Comment:
Truth doesn't seem to add anything compared to assertEquals in this method, and bloats the code a bit.